parm/archive/gefs_extracted_ocean.yaml.j2

```
@@ -0,0 +1,36 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".ocean.t" + cycle_HH + "z." %}
+
+gefs_ocean:
+    name: "GEFS_OCEAN"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_ocean.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set members = [] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do members.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% set res = (OCNRES|string())[0] ~ "p" ~ (OCNRES|string())[-2:] %}
+
+{% for mem in members %}
+    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                          '${RUN}':RUN,
+                          '${YMD}':cycle_YMD,
+                          '${HH}':cycle_HH,
+                          '${GRID}':res,
+                          '${MEMDIR}':mem }) %}
+
+    {% set COMIN_OCEAN_NETCDF = COM_OCEAN_NETCDF_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+    # Select netcdf files to copy to the atardir
+    {% if path_exists(COMIN_OCEAN_NETCDF) %}
+        {% for fhr in range(FHMIN_GFS + FHOUT_OCN_GFS, FHMAX_GFS + FHOUT_OCN_GFS, FHOUT_OCN_GFS) %}
+            {% set file_name = head ~ res ~ ".f" ~ '%03d'|format(fhr) ~ ".nc" %}
+            {% set file_path = COMIN_OCEAN_NETCDF ~ "/" ~ file_name %}
+            - "{{ file_path | relpath(ROTDIR)}}"
+        {% endfor %}
+    {% endif %}
```

`COM_OCEAN_NETCDF_TMPL` should be used here and `COM_OCEAN_NETCDF_GRID_TMPL` should be removed from `config.com`.

```suggestion
    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
                          '${RUN}':RUN,
                          '${YMD}':cycle_YMD,
                          '${HH}':cycle_HH,
                          '${MEMDIR}':mem }) %}

    {% set COMIN_OCEAN_NETCDF = COM_OCEAN_NETCDF_TMPL | replace_tmpl(tmpl_dict) %}

    # Select netcdf files to copy to the atardir
    {% set netcdf_grid_dir = COMIN_OCEAN_NETCDF ~ "/" ~ res %}
    {% if path_exists(netcdf_grid_dir) %}
        {% for fhr in range(FHMIN_GFS + FHOUT_OCN_GFS, FHMAX_GFS + FHOUT_OCN_GFS, FHOUT_OCN_GFS) %}
            {% set file_name = head ~ res ~ ".f" ~ '%03d'|format(fhr) ~ ".nc" %}
            {% set file_path = netcdf_grid_dir ~ "/" ~ file_name %}
            - "{{ file_path | relpath(ROTDIR)}}"
        {% endfor %}
    {% endif %}
```